server/src/main/java/org/elasticsearch/index/translog/TranslogDeletionPolicy.java

```diff
@@ -23,8 +23,8 @@ public final class TranslogDeletionPolicy {
 
     private final Map<Object, RuntimeException> openTranslogRef;
 
-    public void assertNoOpenTranslogRefs() {
-        if (openTranslogRef.isEmpty() == false) {
+    public void ensureNoOpenTranslogRefs() {
```

Comment:
nit, I might have kept the old name, since this function does nothing if assertions are not enabled?

Reply:
Yeah, keeping the original name makes sense to me too.

Reply:
Ok, I reverted that.